qbft/spectest/tests/controller_spectest.go

```diff
@@ -278,4 +281,6 @@ func NewControllerSpecTest(name string, documentation string, runInstanceData []
 		StartHeight:     startHeight,
 		PrivateKeys:     testingutils.BuildPrivateKeyInfo(privateKeys),
 	}
+	test.Controller = test.generateController()
```

Comment:
I think this is a bit funny that the start height and the controller are seperate...
As long as it is fine with SigP...